### DISCUSSION TITLE
Add group sorting to tablist

### DISCUSSION THREAD

**Discussion by Mrkillero4ka** [Zrips/CMI#7695] [Enhancement]
It seems like such a big CMI plugin, but there is no group sorting in it

Written in detail:
Add the ability to the **tablist** function so that in the tab you can make a list group filter, so that, for example, the player has the default group and after it comes silver on top, and after silvera above goes master and so on

**Comment by CloudeLecaw** [comment]
XDDDDD

![image](https://user-images.githubusercontent.com/12080017/227136822-21f20828-b027-44c0-9d1d-6832bec91e37.png)
Might would be better to add some comments in one of the other allready existing discussions to this theme instead of making a new request. there are a lot allready about tab sorting of any kinds.

Tab Sorting is the most wanted and most asked one since over 3 years now. One problem also is: people mostly just open a new request may because they think this idea is new?... i dont know..  but you see its no new idea. but if everyone opens new threads instead of discussion in existing one, it could be this wantage never got heared.

here a little collection where players who ask for tablist grouping of any kind as a new feature of cmi.

https://github.com/Zrips/CMI/discussions/470

https://github.com/Zrips/CMI/discussions/653

https://github.com/Zrips/CMI/discussions/1480

https://github.com/Zrips/CMI/discussions/1492

https://github.com/Zrips/CMI/discussions/2474

https://github.com/Zrips/CMI/discussions/2671

https://github.com/Zrips/CMI/discussions/3479

https://github.com/Zrips/CMI/discussions/3571

https://github.com/Zrips/CMI/discussions/4197

https://github.com/Zrips/CMI/discussions/4695

https://github.com/Zrips/CMI/discussions/5233

https://github.com/Zrips/CMI/discussions/5376

https://github.com/Zrips/CMI/discussions/5769

https://github.com/Zrips/CMI/discussions/5948

https://github.com/Zrips/CMI/discussions/6031

https://github.com/Zrips/CMI/discussions/6388

ohh an now its one more :-D

https://github.com/Zrips/CMI/discussions/7695

**Reply by Mrkillero4ka** [reply]
If there are already so many requests, then why is it still not added?

**Reply by CloudeLecaw** [reply]
> If there are already so many requests, then why is it still not added?

because its not up to us to add it. Its up to Zrips. How he feel, if he wanne add it or not or if he has time to add something there

**Reply by Mrkillero4ka** [reply]
Well, since so many people are asking, then you can add. I think Zrips will be able to do it in 10 minutes, since he is an experienced programmer

**Reply by CloudeLecaw** [reply]
> Well, since so many people are asking, then you can add. I think Zrips will be able to do it in 10 minutes, since he is an experienced programmer

maybe :D  well we wait since 3 years for it. ;) noone knows why. but yes your right... many many people asking for it :D

**Reply by mrfloris** [reply]
> If there are already so many requests, then why is it still not added?

Just because people want it, doesn't mean it will get added, that there's time to add it, or that it can get added without compromise. 

Hopefully one day. But please next time, check for existing threads, so you can contribute to those with your needs, instead of creating another thread again. 

Group sorting in tablist and bungee support for tablist, is one of the most requested tablist feature, so hopefully one day.

**Comment by mrfloris** [comment]
Tablist.yml has player-group sorting in the latest builds of cmilib and cmi. Upgrade and review tablist.yml's configuration (bottom of page). You can sort ascending or descending by type, and types range from permission node weight to baltop rank, etc.
